include/experimental/__p0009_bits/mdspan.hpp

```diff
@@ -111,76 +115,113 @@ class mdspan
   //--------------------------------------------------------------------------------
   // [mdspan.basic.cons], mdspan constructors, assignment, and destructor
 
+#if !MDSPAN_HAS_CXX_20
   MDSPAN_INLINE_FUNCTION_DEFAULTED constexpr mdspan() = default;
+#else
```

Comment:
Per previous discussion: Backporting the requirements to pre-C++20 would likely require duplicating and specializing the entire `mdspan` class.  That wouldn't be worth the benefit (since the class wouldn't compile anyway if the requirements weren't met).